site/docs/windows.md

```diff
@@ -184,7 +184,14 @@ To build C++ targets, you need:
     SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk).
 
     The Windows SDK contains header files and libraries you need when building
-    Windows applications, including Bazel itself.
+    Windows applications, including Bazel itself. You can specify Windows SDK version
+    by setting `BAZEL_WINSDK_FULL_VERSION`. You can use a full Windows 10 SDK number such as 
+    10.0.10240.0, or specify 8.1 to use the Windows 8.1 SDK. Please make sure you have the specified
```

Comment:
> or specify 8.1 to use the Windows 8.1 SDK

This suggests 8.1 is the only acceptable value for the Windows 8.1 SDK. What I guess you meant is:

> or specify a Windows OS version to use the latest SDK installed for that (e.g. 8.1 to use the latest installed Windows 8.1 SDK)

Reply:
That's actually I meant, there's no subversion for Windows 8.1 SDK, only one version is available. I copied this from the official documentation. ;)
https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=vs-2019

Reply:
I see, thanks! Could you maybe call this out in the doc explicitly?

Reply:
Done.